bindata/network/multus/multus.yaml

```diff
@@ -874,5 +924,174 @@ spec:
             items:
             - key: whereabouts.conf
               path: whereabouts.conf
+---
+apiVersion: apps/v1
+kind: Daemonset
+metadata:
+  name: whereabouts-watcher
```

Comment:
let's rename this `whereabouts-token-watcher` -- as we have some `token-watcher` nomenclature upstream (and maybe we can directly use that script here, in follow up), and it'll be more specific